src/cmd/cli/command/servicemonitor.go

```diff
@@ -29,27 +27,27 @@ func waitServiceState(ctx context.Context, targetState defangv1.ServiceState, se
 
 	// monitor for when all services are completed to end this command
 	for newStatus := range subscribeServiceStatusChan {
+		term.Debugf("service %s with state ( %s ) and status: %s\n", newStatus.Name, newStatus.State, newStatus.Status)
+
 		if _, ok := serviceState[newStatus.Name]; !ok {
 			term.Debugf("unexpected service %s update", newStatus.Name)
 			continue
 		}
 
 		// exit on detecting a FAILED state
 		if newStatus.State == defangv1.ServiceState_SERVICE_FAILED {
-			return ErrDeploymentFailed
+			return errDeploymentFailed
 		}
 
 		serviceState[newStatus.Name] = newStatus.State
 
 		if allInState(targetState, serviceState) {
-			for _, sInfo := range serviceInfos {
```

Comment:
this was unused, so I removed it

Reply:
Added back to commands.go, as the value is used in `printEndpoints`